sql/sql_yacc.yy

```diff
@@ -8009,6 +8011,27 @@ alter:
           }
         ;
 
+opt_account_options:
+        /* Nothing */ {}
+        | opt_account_options_list
+        ;
+
+opt_account_options_list:
```

Comment:
eh, why did you allow a sequence of `ACCOUNT LOCK` and `ACCOUNT UNLOCK` in one statement? Specifying it just once isn't enough?